### PR TITLE
Update patient summary header UI

### PR DESCRIPTION
**Pull request by vinaysshenoy**
Pivotal Story Link: https://www.pivotaltracker.com/story/show/161577651

This needs to be done for showing the new Simple ID  once its associated to the patient.

### CODE REVIEW

**Comment by vinaysshenoy** [comment]
@saket The collapsing part was trickier than I expected since what we want to achieve is not supported out of the box. I have decided to do the collapsible part as a separate story (https://www.pivotaltracker.com/story/show/164460711) and leave this one as is and complete the simple passport feature first.

**Comment by saket** [comment]
@vinaysshenoy there's one failing test. Please check?

**Comment by vinaysshenoy** [comment]
> @vinaysshenoy there's one failing test. Please check?

Seems like the android tests step failed on Bitrise again. :(

**Comment by vinaysshenoy** [comment]
> > @vinaysshenoy there's one failing test. Please check?
> 
> Seems like the android tests step failed on Bitrise again. :(

@saket I ran the build again. It passed.

File: app/src/main/res/layout/screen_patient_summary.xml [hunk 1]
```diff
@@ -79,9 +45,43 @@
...
+      android:layout_below="@id/patientsummary_address"
+      android:layout_marginTop="@dimen/spacing_4"
+      android:drawablePadding="@dimen/spacing_4"
+      android:drawableStart="@drawable/ic_call_16dp"
+      android:drawableTint="@color/white72"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Left.Numeric.White72"
+      android:visibility="gone"
+      tools:text="1111111111"
+      tools:visibility="visible" />
+  </RelativeLayout>
```
Comment (saket) [inline app/src/main/res/layout/screen_patient_summary.xml]: @vinaysshenoy the spacings don't match with Zeplin. Can you check?

Comment (saket) [reply app/src/main/res/layout/screen_patient_summary.xml]: Everything else looks good. Let me know once you've fixed this.

Comment (vinaysshenoy) [reply app/src/main/res/layout/screen_patient_summary.xml]: Done.

**Pull request merged**